crates/telio-nurse/src/qos.rs

```diff
@@ -189,6 +190,13 @@ impl Runtime for Analytics {
                 })
             },
 
+            // Manual trigger
+            Ok(_) = self.io.manual_trigger_channel.recv(), if self.ping_channel_tx.upgrade().is_none() => {
```

Comment:
so it's more like manual ping trigger not, qos trigger.

Reply:
It still collects statistics, so it is kind of qos trigger.